src/src/hooks/SelfServiceApi.js

```diff
@@ -34,31 +36,33 @@ export function useSelfServiceRequest() {
                 const newData = await httpResponse.json();
                 setResponseData(newData);
             } else {
-                setErrorMessage("Oh no! We had an issue while retrieving data from the api. Please reload the page.");
-                // if (httpResponse.headers.get("Content-Type") === "application/problem+json") {
-                //     const { detail } = await httpResponse.json();
-                //     setErrorMessage(detail);
-                // } else {
-                //     setErrorMessage("Oh no! We had an issue while retrieving data from the api. Please reload the page.");
-                // }
+                triggerError(NewErrorTriggerRequestBuilder()
+                  .setHttpResponse(httpResponse)
+                  // .setHandler((params) => {
+                  //   if (params.httpResponse.headers.get("Content-Type") === "application/problem+json") {
+                  //       const { detail } = await httpResponse.json();
+                  //       setErrorMessage(detail);
+                  //   } else {
+                  //       setErrorMessage("Oh no! We had an issue while retrieving data from the api. Please reload the page.");
+                  //   }
+                  // })
```

Comment:
should we delete these lines?

Reply:
Removed in https://github.com/dfds/selfservice-portal/pull/25/commits/baed7bb34952f2db0f9fa11fd79886a7ba915e48